Orange/widgets/visualize/owscatterplotgraph.py

```diff
@@ -91,6 +91,31 @@ def clear(self):
 
         self.updateSize()
 
+    def pen(self):
+        if self.__pen is not None:
+            return QPen(self.__pen)
+        else:
+            color = self.palette().color(QPalette.Disabled, QPalette.Text)
+            color.setAlpha(100)
+            pen = QPen(color, 1)
+            pen.setCosmetic(True)
+            return pen
+
+    def brush(self):
+        if self.__brush is not None:
+            return QBrush(self.__brush)
+        else:
+            color = self.palette().color(QPalette.Window)
```

Comment:
I think this is too dark, at least on my machine. I don't have a better suggestion, except, perhaps, `super().brush()` (which, for me, apparently uses a transparent background).

<img width="770" alt="Screenshot 2021-11-19 at 14 04 23" src="https://user-images.githubusercontent.com/2387315/142629367-8844a178-7425-4780-8109-8cf2d4512c03.png">

Reply:
Strange its supposed to be: 
<img width="789" alt="Screenshot 2021-11-19 at 14 29 02" src="https://user-images.githubusercontent.com/4716745/142630599-654ba4ee-6693-4b37-8db8-36a107e00509.png">

Reply:
I think I can finally say: Janez, your Qt is a mess. 😆 
(i.e. My version is ok, too.)

Reply:
My `QPalette.Window` is black, which it indeed shouldn't be (in light mode on macOS). Any ideas why?

```
>>> hex(QPalette().color(QPalette.Window).rgb())
'0xff000000'
```

Reply:
Can you post the contents of `~/.config/biolab.si/Orange.ini`in particular `[application-style]` section.

Reply:
```
[application-style]
palette=
style-name=
```